tests/e2e/translations/nepali-dates-and-numbers.wdio-spec.js

```diff
@@ -165,5 +227,50 @@ describe('Bikram Sambat date display', () => {
     expect(await contactsPage.getContactSummaryField('phone')).to.equal('+४०७५५४५६४५६');
     expect(await contactsPage.getContactSummaryField('field')).to.equal('text ०१२३४५६७८९');
     expect(await contactsPage.getContactSummaryField('another')).to.equal('other text 0123456789');
+  });  
+
+  it('SMS report shows bsDate type as date field correctly', async () => {
+    await setLanguage(NEPALI_LOCALE_CODE);
+    moment.locale(NEPALI_LOCALE_CODE);
+
+    await gatewayApiUtils.api.postMessage({
+      id: 'lmp-id-bs',
+      from: '+9779876543210',
+      content: `${formIdBS} Shrestha ${momentToBikYMD(tenWeeksAgo)}`
+    });
+
+    await commonPage.goToPeople();
+    await commonPage.goToReports();
+    const firstReport = reportsPage.firstReport();
+    firstReport.click();
+
+    const dateFormat = bikramSambat.toBik_text(tenWeeksAgo);
+    const relativeFormat = moment(tenWeeksAgo.toDate()).fromNow();
+    const lmpDateValue = await reportsPage.getReportDetailFieldValueByLabel('LMP Date');
+    expect(lmpDateValue).to.equal(`${dateFormat} (${relativeFormat})`);
+  });
+
+  it('SMS report shows bsAggreDate type as date field correctly', async () => {
+    await setLanguage(NEPALI_LOCALE_CODE);
+    moment.locale(NEPALI_LOCALE_CODE);
+
+    await gatewayApiUtils.api.postMessage({
+      id: 'lmp-id-bs-parts',
+      from: '+9779876543210',
+      content: `${formIdBSParts} Shrestha ` +
+      `${momentToBikParts(nineWeeksAgo).year} ` +
+      `${momentToBikParts(nineWeeksAgo).month} ` +
+      `${momentToBikParts(nineWeeksAgo).day}`
+    });
+
+    await commonPage.goToPeople();
+    await commonPage.goToReports();
+    const firstReport = reportsPage.firstReport();
```

Comment:
I think both of these need to be awaited.

Reply:
Added `await` but reverted because of error.

Reply:
I tried with both await and no await but there seems to be no difference, so removed it.

Reply:
There is a difference, in the sense that without awaiting you can risk making the test flaky, because of race conditions.

Reply:
Thanks for the clarification, I've added it. :)